### ISSUE TITLE
lower bound of Cabal dependency is wrong

### ISSUE THREAD

**Issue by osa1** [jberthold/packman#1]
`Distribution.Simple.Test.LibV09` is added with 1.20.0.0, so `>= 1.18` is not a correct lower bound.

Fixing that is still causing problems though. Pre-compiled GHC 7.8.3 binaries are linked with `Cabal-1.18.1.3`. So there's no way to compile test suite using pre-compiled GHC 7.8.3.

EDIT: I just realized that `LibV09` is not directly referenced from `packman`. It's in the generated code `dist/build/alltestsStub/alltestsStub-tmp/alltestsStub.hs`. I think this is generated by `cabal-install`, which is newer than the version used to compile `packman` (because `packman` uses GHC and I installed it from pre-compiled binaries, which are linked with 1.18)

So in short, test program generation is done by system-wide installed cabal, but compilation of generated code is done using the version in the sandbox, which is 1.18.3, GHC requires that.

I'm closing this as this has nothing to do with `packman`. Sorry for the noise.
